boards/m5stack/m5stack_cores3/Kconfig.m5stack_cores3

```diff
@@ -0,0 +1,9 @@
+# M5Stack CoreS3 board configuration
+
+# Copyright (c) 2024 Zhang Xingtao <zhxt@live.cn>
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_M5STACK_CORES3
+	select SOC_ESP32S3_WROOM_N16R8
```

Comment:
The CoreS3 is not implementing WROOM module.
(But it is functionally identical, so there is no problem about this point.)
It should stand on the bare SoC definition.

Please consider applying https://github.com/zephyrproject-rtos/zephyr/pull/79765/commits/32435b1c5047d65dfebc85af9f5af3fabd0e4976

Reply:
> The CoreS3 is not implementing WROOM module. (But it is functionally identical, so there is no problem about this point.) It should stand on the bare SoC definition.
> 
> Please consider applying [32435b1](https://github.com/zephyrproject-rtos/zephyr/commit/32435b1c5047d65dfebc85af9f5af3fabd0e4976)

Yeah, you're right!  Sorry for the request overwrote. Could you please re-submit it. Many thanks!

Reply:
> Yeah, you're right! Sorry for the request overwrote. Could you please re-submit it. Many thanks!

I think it probably can't be overwritten unless I delete it.

https://github.com/zephyrproject-rtos/zephyr/commit/32435b1c5047d65dfebc85af9f5af3fabd0e4976

If you can't apply it as is, please use it as a reference and make a correction.